docs/API.md

```diff
@@ -3079,7 +3079,7 @@ __Parameters__
 
 |Return Type	  | Exceptions	  |
 |:--- |:--- |
-| ``Iterable<Result<DeleteError>>``:an iterator of Result DeleteError.  | _None_  |
+| ``Iterable<Result<DeleteError>>``:a lazy iterator of Result DeleteError.  | _None_  |
```

Comment:
I don't think this change is needed as the description should carry about lay eval.